cmd/2019_2/main.go

```diff
@@ -46,12 +53,18 @@ func main() {
 
 	flag.Parse()
 
-	instructions := read.Read(flagInput, separator)
-	instructions2 := make([]int, len(instructions))
-	copy(instructions2, instructions)
+	var (
+		instructions  = read.Read(flagInput, separator)
+		instructions2 = make([]int, len(instructions))
+		outputs       = make(chan int, 1)
+	)
 
+	copy(instructions2, instructions)
 	instructions[1] = instruction1
 	instructions[2] = instruction2
-	utils.AssertEqual(intcode.Compute(instructions, intcode.Multiply), resCompute)
+
+	intcode.Compute(instructions, intcode.Multiply, make(chan int, 1), outputs)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '1', create a named constant for it](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)